### ISSUE TITLE
Exponential backoff int64 overflow

### ISSUE THREAD

**Issue by jcageman** [Azure/azure-functions-host#4699]
The following line in ComputeBackoff can cause a Int64 overflow:
https://github.com/Azure/azure-functions-host/blob/b44799489586e1912877160e3cfee2a276463a37/src/WebJobs.Script/Utility.cs#L123

So far i only noticed it locally, but it can probably happen on azure itself as well!

**Comment by fabiocav** [comment]
Indeed. Curious, did you actually run into an exception with this code path? Do you have the stack for this? There are two main code paths that use this, host restarts and the file watcher recovery. Curious to understand which one caused the problem (though, both are exposed).
